<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color:white; color:black;}QLabel#Company_Name{font-size: 32px; font-family: georgia;} 
QLineEdit{font-size:20px}
QLabel#subtitle{font-family:georgia}
QPushButton#home_button{text-decoration:underline;color:blue}
QLabel#home_label{font-size:15px;}
QLabel#transaction_label{font-size:20px;color: rgb(200, 0, 0)}
QTextEdit#transaction_history_label{font-size:20px;}
QLabel#withdraw_label{font-size:20px;}
QLabel#welcome_label_2{font-size:15px;color: rgb(200, 0, 0); text-decoration:underline}
QLabel#response_message{color: rgb(200, 0, 0)}
QPushButton#withdraw_button_2{background-color: rgb(220,220,220)}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>depositphotos_81119782-stock-photo-bank-flat-red-color-round.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="Company_Name">
           <property name="text">
            <string>PyBanking</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="subtitle">
           <property name="text">
            <string>Where banking is a pleasure</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="welcome_label_2">
       <property name="text">
        <string>Welcome USER</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="transaction_label">
         <property name="text">
          <string>Transactions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="transaction_history_label"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="home_label">
       <property name="text">
        <string>Would you like to return to the home page?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="home_button">
       <property name="text">
        <string>Go Back</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
